Dockerfile

```diff
@@ -10,7 +10,7 @@ USER root
 RUN apt-get update \
     && apt-get install --no-install-recommends -y \
     bzip2=1.0.6-8.1 \
-    patch=2.7.5-1+deb9u1 \
+    patch=2.7.5-1+deb9u2 \
     libfontconfig1=2.11.0-6.7+b1 \
```

Comment:
This change has been added to fix the build.  This small change come from master. Currently development has not got this change.